frontend/src/Components/UserProfileInvites.tsx

```diff
@@ -46,8 +46,8 @@ export const UserProfileInvites = () => {
                 ?
                     <>
                         {activeInvites.map(invite =>
-                            <>
-                                <div key={invite.code} className='item'>
+                            <React.Fragment key={invite.code}>
```

Comment:
Why do we even wrap the div in the fragment? I think fragment can be removed and key added to the div.

Reply:
makes sense, removed